src/simulation/dynamics/spinningBodies/spinningBodiesTwoDOF/spinningBodyTwoDOFStateEffector.cpp

```diff
@@ -37,6 +37,20 @@ SpinningBodyTwoDOFStateEffector::SpinningBodyTwoDOFStateEffector()
     this->IS2PntSc2_S2.setIdentity();
     this->dcm_S10B.setIdentity();
     this->dcm_S20S1.setIdentity();
+    this->ATheta.setIdentity();
```

Comment:
Here, I think we can use ``setZero()`` and leave ``setIdentity()`` to the DCMs and inertias (lines 47-51).

Reply:
For sure! Done.